backend/tracim_backend/command/__init__.py

```diff
@@ -82,6 +95,14 @@ def _setup_logging(self, parsed_args: Namespace) -> None:
     def get_parser(self, prog_name: str) -> argparse.ArgumentParser:
         parser = super(AppContextCommand, self).get_parser(prog_name)
 
+        parser.add_argument(
```

Comment:
Previously tested this (in several manners) and in different places of the code and it would always break the tests at some point, because it limits several actions related to table creation and such.

I'm pretty sure *there is* a proper solution, but I don't think this is really worth the time at the moment since even if sub-optimal adding a manual flag works. I mean, if the PR is merged as it is, it would be necessary to create an issue dedicated to this problem.

Reply:
Sure !